src/userinput.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _USERINPUT_H
```

Comment:
We need a copyright + license header in every file to comply with GPLv3 and beeing able to package openfortivpn for Fedora, Debian, etc.